cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.1 on 2023-12-08 00:18
+
+import cinema.models
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):

```

Comment:
You don't have to push migrations to your branch. Use `git rm -cached cinema/migrations -r` to remove it.